section2/exercises/if-statements.js

```diff
@@ -86,11 +99,15 @@ var cupsOfFlour = 1;
 var hasSauce = true;
 
 // Write your conditional here
-
+if (cupsOfFlour >=2 && hasSauce === true) {
```

Comment:
Really nice and clean solution 🚀

If you want to have fun... remove the `=== true` portion of line 102 and see what happens/explore why.